cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/LowestAddressJoinDecider.scala

```diff
@@ -137,7 +137,7 @@ class LowestAddressJoinDecider(system: ActorSystem, settings: ClusterBootstrapSe
 
       // some discovery mechanism can return both host name and IP address. this checks for both.
       def hostMatches(host: String, lowest: ResolvedTarget): Boolean =
-        (Some(host) == lowest.address) || (host == lowest.host)
+        (Some(host) == lowest.address.map(_.getHostAddress)) || (host == lowest.host)
```

Comment:
since now this has an op let's inverse them

`(host == lowest.host) || ...`

Reply:
ok.

Reply:
done c5b633036f229e172c064fb370e35b2a7762cf7b